ccm_web/server/src/invitation/cirrus-invitation.interfaces.ts

```diff
@@ -0,0 +1,26 @@
+import { hasKeys } from '../typeUtils'
+
+export const apiURL = 'https://apps.cirrusidentity.com/console'
```

Comment:
@lsloan I don't want this hardcoded in the code. I still want this as configurable value, coming from `.env` file. I feel this needs to change

Reply:
🏁